docs/content/styles/fonts.mdx

```diff
@@ -66,5 +65,14 @@ const customStyles = `
   }
 `;
 
-globalStyles({ theme: yourTheme, custom });
+export default function App() {
+  return (
+    <ThemeProvider theme={theme.circuit}>
+      <GlobalStyles custom={customStyles} />
+      <p>
+        Hello World!
+      </p>
+    </ThemeProvider>
+  );
+}
```

Comment:
@connor-baer Is this accurate? I made it the same as how we are using it, but I'm not sure if it complies with the font-loading strategy you described above?

Reply:
Looks accurate to me. You only missed to update `globalStyles` in [L51](https://github.com/sumup/circuit-ui/pull/426/files/bd312a2b382cbfca77f2603033259becdb546f30#diff-9611e01837e15359f4de3ae3a3cf59beR51)

Reply:
Will take care of that, thanks for your eagle eyes 🦅